page.php

```diff
@@ -1,36 +1,44 @@
 <?php get_header(); ?>
 
-	<div class='main-section'>
-	  <ul class="posts">
-		<?php 
-		  if ( have_posts() ) :
-	      while ( have_posts() ) : the_post();
-			get_template_part( 'content', 'page' );
-		?>
-
-		<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-
-		<?php the_title('<h1 class="entry-title">','</h1>' ); ?>
-
-		<?php if( has_post_thumbnail() ): ?>
-			<div class="thumbnails">
-				<?php the_post_thumbnail('thumbnail'); ?>
-			</div>
-		<?php endif; ?>
-
-		<small>
-			<?php the_category(); ?>
-		</small>
-
-		<div>
-		  <?php the_content(); ?>
-		</div>
-
-		<?php 
-			endwhile;
-			endif;
-		?>
-	  </ul>
-	</div>
+  <div class='main-section'>
+    <?php 
+      if ( have_posts() ) :
+    ?>
+        <ul class="posts">
```

Comment:
It's a page, so you'll only have one post, not a list of posts. This means your ul is unnecessary.